paasta_tools/config_utils.py

```diff
@@ -28,6 +28,11 @@
     "cassandracluster",
 )
 
+# this could use a better name - but basically, this is for pairs of instance types
+# where you generally want to check both types (i.e.,g a status-quo and migration
+# instance type)
+INSTANCE_TYPE_COUNTERPARTS = {"eks": "kubernetes", "kubernetes": "eks"}
```

Comment:
Just to make sure I'm following, we probably won't ever be using this first item until we start having the CSV spit out 'eks-*' instance_type_cluster results, right?

Reply:
yea - although I'm hoping that we never get to that point and simply s/eks-/kubernetes-/ all the soaconfigs :p

but, better safe than sorry in case we keep the eks files for a bit